pyvodb/load.py

```diff
@@ -53,21 +53,30 @@ def dict_from_directory(directory, root, ignored_files=()):
     return data
 
 
+def load_meta_file(directory):
+        metafile = os.path.join(directory, 'meta.yaml')
+        with open(metafile) as f:
+            info = yaml.load(f, Loader=YAML_SAFE_LOADER)
+        info['ignored_files'].append('meta.yaml')
+        return info
+
+
 def load_from_directory(db, directory):
+    metadata = load_meta_file(directory)
     data = dict_from_directory(
-        '.', directory, ignored_files=['.git', 'README', 'tests'])
-    load_from_dict(db, data)
+        '.', directory, ignored_files=metadata['ignored_files'])
```

Comment:
Can this be `ignored_files=metadata.get('ignored_files', ['.git', 'README', 'tests'])`, for backwards compatibility?